### PR TITLE
Deployment setup + sound effects

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
## Description
- Ready for deployment
- Added sound effects

### CODE REVIEW

File: src/components/Call/Call.tsx [hunk 1]
```diff
@@ -122,13 +128,15 @@ const CallBase: React.FC<Props> = React.memo(
     }, [peerAudioOn, call, participantHasJoined, t]);
 
     useEffect(() => {
-      if (participantHasJoined && call)
+      if (participantHasJoined && call) {
+        playJoinCall();
```
Comment (mpekala23) [inline src/components/Call/Call.tsx:132]: nice this is very clean

File: src/components/Dashboard/CallItem.tsx [hunk 2]
```diff
@@ -20,12 +24,16 @@ const CallItem: React.FC<Props> = ({
 }: Props) => {
   const { t } = useTranslation("dashboard");
 
+  const [play] = useSound(EnterCallSound);
```
Comment (mpekala23) [inline src/components/Dashboard/CallItem.tsx:27]: maybe a slightly more descriptive name like with the sounds before? playEnterSound or something?

File: src/pages/Call/index.tsx [hunk 3]
```diff
@@ -50,15 +49,15 @@ const CallBase: React.FC<RouteComponentProps<TParams>> = ({ match }) => {
...
+    dispatch(
       initializeVisit({
         callId: call.id,
         setRc,
         authInfo,
         videoHandler: call.videoHandler,
       })
     );
-  }, [call, stableDispatch, hasInit, authInfo]);
+  }, [call, hasInit, authInfo]);
```
Comment (mpekala23) [inline src/pages/Call/index.tsx:60]: 👍

File: src/pages/Login/index.tsx [hunk 4]
```diff
@@ -76,6 +76,7 @@ function LoginContainer({ session }: PropsFromRedux): ReactElement {
         language: values.language,
       });
     } catch (err) {
+      console.log(err);
```
Comment (mpekala23) [inline src/pages/Login/index.tsx:79]: ooop

File: src/redux/modules/call.ts [hunk 5]
```diff
@@ -37,9 +37,10 @@ export const initializeVisit = createAsyncThunk(
     setRc: (rc: RoomClient) => void;
     videoHandler: CallHandler;
   }) => {
-    console.log(`https://${videoHandler.host}:${videoHandler.port}`);
     const socket = io.connect(
-      `https://${videoHandler.host}:${videoHandler.port}`,
+      process.env.NODE_ENV === "production"
+        ? `https://${videoHandler.host}`
+        : `https://${videoHandler.host}:${videoHandler.port}`,
```
Comment (mpekala23) [inline src/redux/modules/call.ts:43]: do we not need a port in production?

**Review by mpekala23 — APPROVED** [review]
few quick nits but looks good!

File: src/redux/modules/call.ts (hunk 5 above)
Comment (gabrielsaruhashi) [reply src/redux/modules/call.ts:43]: nope!

File: src/components/Dashboard/CallItem.tsx (hunk 2 above)
Comment (gabrielsaruhashi) [reply src/components/Dashboard/CallItem.tsx:27]: will do a fast follow

**Pull request merged**